### ISSUE TITLE
should report: missing space after keyword

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#537]
```abap
DATA foo TYPE string.
CASE foo.
  WHEN'bar'.
ENDCASE.
```
